Sources/LabelAnalysis.swift

```diff
@@ -85,8 +85,9 @@ class LabelAnalysis {
   /// submitter of the change.
   ///
   /// - Parameter issueData: The incoming issue data
-  class func addAndFixLabelsForIssues(issueData: IssueData) {
-    let componentNames = GithubAPI.getDirectoryContentPathNames(relativePath: "components")
+  class func addAndFixLabelsForIssues(issueData: IssueData, githubInstance: GithubAPI) {
```

Comment:
What do you think if we refer to this as `githubAPI`? I'm not sure "Instance" is communicating anything to me for this variable.

Reply:
sounds fair, updated the naming, thanks!